score/ipython/Score_Dissolve_Script.ipynb

```diff
@@ -0,0 +1,454 @@
+    "CENSUS_GEOJSON_DIR = DATA_DIR / \"census\" / \"geojson\"\n",
+    "CEJST_DATA_PATH = DATA_DIR / \"score\" / \"csv\" / \"usa.csv\"\n",
+    "score_df = pd.read_csv(CEJST_DATA_PATH, dtype={\"GEOID10\": \"object\"})"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "Ia5bqxS2LJqe"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "master_df = gpd.GeoDataFrame()"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "Dtf5qD50JvCw"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    " for file_name in CENSUS_GEOJSON_DIR.rglob('*.json'):\n",
+    "    state_gdf = gpd.read_file(file_name)\n",
+    "    master_df = master_df.append(state_gdf)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "PNdw8bERJyKk"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "master_df = master_df.to_crs(\"+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "B5SS9y2pLwks"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "master_df.shape"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "_C6vaR9HQeLa",
+    "outputId": "fab3bc7f-e716-431e-bc76-bd26289ea4a4"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "master_df.head(2)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "oMoubjqCQiw5",
+    "outputId": "6195ffbc-6275-40c6-bb6a-e0a6bd1e71f0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_df = pd.read_csv(CEJST_DATA_PATH, dtype={\"GEOID10\": \"object\"})"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "bAMmGSgzVml0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged = master_df.merge(usa_df, on=\"GEOID10\", how=\"left\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "U7M7dExdV2Vh"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged.head(2)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "Sr25DUkxWVhg",
+    "outputId": "1e804075-0f7d-4174-82d7-e21b8519c8bf"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged_compressed = gpd.GeoDataFrame(usa_merged, crs=\"EPSG:4326\")"
+   ],
+   "outputs": [],
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "ANMlAB8Qmtu8",
+    "outputId": "44934741-90a9-4664-fab5-2c39b348d2be"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "usa_merged_compressed.to_file(\"usa_merged.geojson\", driver=\"GeoJSON\")"
```

Comment:
Why is this dataframe saved to a file? You need it for the map? I don't see it being read later in the notebook?

Reply:
The thought is we might want to use this for experimentation purposes, e.g. - what if we used the merged dataset for all zoom levels, would perf be that bad, that sort of thing